guides/v2.1/payments-integrations/vault/vault-di.md

```diff
@@ -63,6 +63,7 @@ Example: a gateway command for the authorize action is added for the Braintree v
         <argument name="client" xsi:type="object">Magento\Braintree\Gateway\Http\Client\TransactionSale</argument>
         <argument name="handler" xsi:type="object">BraintreeVaultResponseHandler</argument>
         <argument name="validator" xsi:type="object">Magento\Braintree\Gateway\Validator\ResponseValidator</argument>
+        <argument name="errorMessageMapper" xsi:type="object">Magento\Braintree\Gateway\ErrorMapper\VirtualErrorMessageMapper</argument>
```

Comment:
Hi @magesanjay 
I don't find an extra argument of Vault Authorization command in [V2.1](https://github.com/magento/magento2/blob/2.1/app/code/Magento/Braintree/etc/di.xml#L248).
Can you please revert change of v2.1 doc files. we can accept these changes for the doc v2.2 and v2.3

Reply:
Hello @ronak2ram 
I have reverted a change in v2.1 and applying it to V2.2. and V2.3.
Thanks

Reply:
thanks, @sanjay for the quick update.